marshmallow/utils.py

```diff
@@ -18,6 +18,9 @@
 
 from marshmallow.compat import binary_type, text_type
 
+ALLOW = "allow"
```

Comment:
@sloria what's your opinion on this? I was just following your API, and I feel it's a great one, but honestly I'm good with both.

Reply:
Also, note to myself: if we keep it that way, I should replace double quotes by single quotes there.

Reply:
@lafrech Using constants prevents typos from passing silently, e.g. `MySchema(unknown='rais')`.

Reply:
The point was to avoid having to import the constant. But I don't mind. In most use cases, this will be setup once in a base `Schema` class for the whole application.